lib/active_fedora/containers/container.rb

```diff
@@ -0,0 +1,28 @@
+module ActiveFedora
+  class Container < ActiveFedora::Base
+
+    property :membership_resource, predicate: ::RDF::Vocab::LDP.membershipResource
+    property :member_relation, predicate: ::RDF::Vocab::LDP.hasMemberRelation
+
+    def parent
+      @parent || raise("Parent hasn't been set on #{self.class}")
+    end
+
+    def parent=(parent)
+      @parent = parent
+      self.membership_resource = [::RDF::URI(parent.uri)]
+    end
+
+    def mint_id
+      "#{id}/#{SecureRandom.uuid}"
+    end
+
+    def self.find_or_initialize(id)
```

Comment:
I'm also :+1: on keeping the API small, but that seems like a motivation to make this method private, or to consider a builder pattern, not to tuck it away in the subclass.  My concern would be that the outcome here is that we end up with an API that is both big *and* inconsistent.